x-pack/legacy/plugins/ml/public/jobs/new_job_new/common/results_loader/searches.ts

```diff
@@ -19,7 +19,7 @@ type TimeStamp = number;
 
 interface Result {
   time: TimeStamp;
-  value: Value;
+  value: unknown;
```

Comment:
@elastic/machine-learning This change was necessary because `Value` seems to have been a global type exposed by Jest, which is no longer exposed after upgrading. I switched to `unknown` because it doesn't seem relevant